test/test.js

```diff
@@ -790,14 +838,29 @@ async function startBrowser(browserName, startUrl) {
     }
   }
 
-  const browser = await puppeteer.launch({
+  const options = {
     product: browserName,
     headless: false,
     defaultViewport: null,
-  });
-  const pages = await browser.pages();
-  const page = pages[0];
-  await page.goto(startUrl, { timeout: 0 });
+  };
+  if (browserName === "chrome") {
```

Comment:
Could you add a comment why these additional Chrome and Firefox preferences are necessary?

Reply:
About option for Chrome, they're required at least on my machine because without them chrome is crashing!
And as a side effect I just discovered that unit tests are running on chrome too.